### DISCUSSION TITLE
Weird behaviour when force rechecking

### DISCUSSION THREAD

**Discussion by Menard01** [arvidn/libtorrent#7587] [Q&A]
libtorrent 2.03 on Linux Mint 21.1
When doing force recheck of some completed torrent, the result is that it is in fact incomplete with often a file at 99% but often less too. 
If I compare the md5 of the file(s) before and after this operation, it is unchanged, so what does it mean ? The recheck is wrong ? 
No because when I am online after this, and that resume is done, a download begin to complete the missing parts... and with peers visible. 
So how can be the download complete and uncomplete at the same time ? 
NB : it is on NTFS partitions

**Comment by Menard01** [comment]
This problem was due to the deletion of the .parts files 
How can you not knowing this ? We must do all by ourselves... and lose time, much time

**Reply by arvidn** [reply]
It sounds like you figured it out, and that it all works as expected. After deleting the partfile, the torrent is no longer complete (but it's probably recorded as complete in the resume data). When you re-check, the missing parts are detected and re-downloaded.

**Comment by yueisme** [comment]
https://github.com/qbittorrent/qBittorrent/issues/19048#issuecomment-2106186508
